### DISCUSSION TITLE
How can I get two listings to display side-by-side?

### DISCUSSION THREAD

**Discussion by ForBo7** [quarto-dev/quarto-cli#2702] [Q&A] [answered]
Hello.

I've been trying to get two listings to be displayed side-by-side on a page.

<img width="1440" alt="Screen Shot 2022-10-02 at 21 12 02" src="https://user-images.githubusercontent.com/85829792/193464490-0f06c07d-ea27-4764-a028-92fea440a93f.png">

I've tried almost everything on the Quarto docs (notably the [Article Layout](https://quarto.org/docs/authoring/article-layout.html) and [Page Layout](https://quarto.org/docs/output-formats/page-layout.html)) and from a few Stack Overflow posts, but I haven't been successful. I've mainly been fiddling with Quarto divs to achieve this.

I would appreciate any suggestions and ideas on how to achieve this!

**Comment by mcanouil** [comment, marked as answer]
A combination of <https://quarto.org/docs/websites/website-listings.html#multiple-listings> and multi-columns layout should do the trick.

You could also use <https://quarto.org/docs/authoring/figures.html#complex-layouts> instead of `.columns`/`.column`.

````
---
title: Team Documents
listing: 
  - id: lab-reports
    contents: "lab-reports/*.qmd"
    type: table
  - id: meeting-notes
    contents: "meeting-notes/*.qmd"
    type: table
---

::::: {.columns}

:::: {.column width="45%"}
## Lab Reports

:::{#lab-reports}
:::
::::

:::: {.column width="45%"}
## Meeting Notes

:::{#meeting-notes}
:::
::::

:::::
````

**Reply by ForBo7** [reply]
Yes, this is exactly what I'm looking for! Thank you so much!
